packages/jaeger-ui/src/components/App/TopNav.tsx

```diff
@@ -29,6 +29,8 @@ import { ConfigMenuItem, ConfigMenuGroup } from '../../types/config';
 import { getConfigValue } from '../../utils/config/get-config';
 import prefixUrl from '../../utils/prefix-url';
 
+import JaegerHomeLink from '../../img/jaeger-home-link.svg';
```

Comment:
what is this for & why didn't we need it before?

Reply:
The upper left link looked like another tab, but the behavior is not the same as in tabs - there is no selected state nor content. We changed it to an image to make it clearer that it’s the ‘home’ link and that it differs from the controls next to it.
![image](https://user-images.githubusercontent.com/3125539/104420433-8a570400-5582-11eb-8e1d-0a6afb81c25a.png)

Reply:
your choice :)
this is how it looks with the logo (personally i don't like it that much):
![image](https://user-images.githubusercontent.com/3125539/104500904-ae98fc00-55e7-11eb-8ca0-645ada71fd10.png)
on the other hand, we can go with a simple text label and a different font, just to distinguish it from the other controls

Reply:
Let's do a text label / different font.